item-spec/examples/sentinel2-sample.json

```diff
@@ -55,6 +55,15 @@
       ]
     ]
   },
+  "properties": {
+    "collection": "s2-l2a",
+    "datetime": "2018-06-05T10:00:27.464Z",
```

Comment:
That datetime (=> microseconds) is invalid, isn't it?

Reply:
You mean that it's .464Z? And should be .46Z? I have no idea, looking at https://tools.ietf.org/html/rfc3339#section-5.6 it says time-secfrac    = "." 1*DIGIT which I think means one or more?

Reply:
While microseconds is theoretically allowed, I see some tools in practice not liking them, so for interoperability and simplicity I generally try to avoid them if possible. I'd propose to change to `2018-06-05T10:00:27Z`.

Reply:
Cool, happy to change it. Should we put that recommendation somewhere in the spec? Could just be in best practices. Though I could see mention in the item spec. Just something like 'only using two decimal places is recommended, since while rfc3339 technically allows microseconds many tools don't understand them.

Reply:
I guess that microsecond thing is quite a bit subjective. I saw several projects trying to get around datetimes with regular expressions and stuff that just doesn't catch microseconds, but I believe most datetime libraries handle it just fine. I think I'd just remove it from examples for simplicity and then see how it goes. We can still add something to BP later if there's really a problem with it.

Reply:
Ok, sounds good.

Reply:
Not sure, but why did you just remove one number from the decimals?